packages/consumption/test/modules/requests/OutgoingRequestsController.test.ts

```diff
@@ -280,6 +280,24 @@ describe("OutgoingRequestsController", function () {
                     "The Request cannot be created because its acceptance would lead to the creation of more than one RelationshipAttribute in the context of this Relationship with the same key 'uniqueKey', owner and value type."
             });
         });
+
+        test("returns a validation result that contains an error for requests that are expired", async function () {
```

Comment:
Please put this test before the `"returns a validation result that contains an error for requests that would lead to the creation of more than one RelationshipAttribute with the same key"` test right above as the Request expiry test is less specific and fits better with the above tests than the Relationship `key` uniqueness test. :D